rbi/rootfs-img/rootfs/.gitignore

```diff
@@ -1,3 +1,4 @@
 kata-Centos-dnf.conf
 rootfs
-report
\ No newline at end of file
+report
```

Comment:
You should write `.gitignore` in the lexicographical order. So please change to

```shell
bin
kata-Centos-dnf.conf
report
rootfs
```

Reply:
deleted useless .gitignore s , and rewrite top-level .gitignore in the lexicographical order.

Reply:
https://github.com/alibaba/inclavare-containers/pull/1255/commits/2f8c530cfc93ded8b7c66fe8e5a87492beac1089